examples/tutorial_pipe_multiprocess.py

```diff
@@ -27,10 +27,10 @@ def run(rank, world_size):
 
     device = torch.device("cuda", RANK) if DEVICE == "cuda" else torch.device("cpu")
 
-    model = fairscale.nn.Pipe(
+    model = MultiProcessPipe(
```

Comment:
Should we specify this separately if it is not an API? Also it might be confusing to understand MultiProcessPipe with the style argument below.

Reply:
I'd like to limit the scope of this change to just separating Pipe from MultiProcessPipe. I plan to focus on cleaning up MultiProcessPipe in subsequent changes. In this change I've limited myself to renaming and dead-code removal.